src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,36 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+type UserWithTodo = {
+  user: User;
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+};
+
+type User = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
```

Comment:
Why do we need specific type for a use in this component ?

Reply:
types were moved from FC to files